src/index.css

```diff
@@ -73,19 +92,20 @@ h1 {
   background-color: #fff;
   box-shadow: 2px 2px 2px #000;
   font-family: "Itim", sans-serif;
-}
-
-#scores {
-  width: 100%;
-  border: #000 solid 3px;
+  transition: all 0.4s;
 }
 
 .record:nth-child(odd) {
   background-color: #ddd;
+  box-shadow: 10px 10px 10px 6px #000;
```

Comment:
[OPTIONAL] I liked your design. However, it looks a bit too much in here when I hover it. I think it would be much better if you could make the border less and its movement to right.

<img width="400" alt="Screen Shot 2022-10-13 at 07 28 37" src="https://user-images.githubusercontent.com/69011963/195485853-14919d2b-8aa6-4357-ac02-7b1f4c1062f3.png">